eggplant_project/settings/base.py

```diff
@@ -15,6 +15,10 @@
 from django.utils.translation import ugettext_lazy as _
 from django.contrib.messages import constants as messages_constants
 
+COOP_NAME = 'KBHFF'
```

Comment:
A design pattern can become a convention if there's a community that promotes it as such... but otherwise a design pattern is just a means to solve a problem that otherwise would be repeated in the codebase... by repeating the same pattern :)

Probably there's other ways, but you're free to call any pattern a "design pattern" until some one argues that it's inferior and actually an anti-pattern :)